deploy/openshift/clusterrole.yaml

```diff
@@ -6,22 +6,40 @@ metadata:
     app: kiali
     version: ${VERSION_LABEL}
 rules:
-- apiGroups: ["", "apps", "autoscaling", "batch"]
+- apiGroups: [""]
   resources:
   - configmaps
-  - cronjobs
-  - deployments
   - endpoints
-  - horizontalpodautoscalers
-  - jobs
   - namespaces
   - nodes
   - pods
   - projects
   - services
+  - replicationcontrollers
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["extensions", "apps"]
```

Comment:
maybe "projects" should fall into that apiGroups?

Reply:
btw do you know how we can get a "topology" of apiGroups & resources registered? This is sometimes confusing in k8s ... objects defined by a CRD are easily trackable, but for others such as "projects" ... I don't know.

Reply:
`kubectl api-resources` lists all API resource types and the apiGroups they belong to.